pom.xml

```diff
@@ -55,18 +127,90 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
I'm not sure it would make a difference, but Lucene 5.5.4 is compatible with Java 7, so maybe we should use Java 7 here? Unless the compiler only actually compiles tests?

See https://lucene.apache.org/core/5_5_4/SYSTEM_REQUIREMENTS.html